holoviews/core/spaces.py

```diff
@@ -509,6 +516,47 @@ def get_nested_streams(dmap):
     return list(set(layer_streams))
 
 
+
+def get_stream_sources(obj, branch=0):
```

Comment:
Any particular reason for the name change from ``get_sources``? Or you just think this is a better name?